src/whiteboard-active.js

```diff
@@ -17,15 +17,14 @@ class WhiteboardActiveAll {
     } else {
       allObjects = virtualclass.wb[wId].canvas.getObjects();
     }
-    
     for (let i = 0; i < allObjects.length; i += 1) {
       console.log('====> active all');
       allObjects[i].set('selectable', true);
     }
   }
 
   disable(wId, component) {
-    let allObjects = component ? virtualclass.wb[wId].canvas.getObjects(component) : virtualclass.wb[wId].canvas.getObjects();
+    const allObjects = component ? virtualclass.wb[wId].canvas.getObjects(component) : virtualclass.wb[wId].canvas.getObjects();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 128. Maximum allowed is 120.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5384064)